### DISCUSSION TITLE
I have a proofs in harvester, but not found in full_node

### DISCUSSION THREAD

**Discussion by weiwenhao** [Chia-Network/chia-blockchain#6632] [Support]
1 plots were eligible for farming abed51d06e... Found 1 proofs. Time: 2.63653 s. Total 222 plots  
harvester deploy -> https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

harvester: chia plots check ↓
WARNING  Plot plot-k32-2021-06-04-17-30-e56b51dd36d2ad91be289ec14b1fc5e93742d39a6baa77fa5c60378a3a289f6c.plot has a pool public key that is not in the farmer's pool pk list.
whether it makes a difference？

**Comment by loppefaaret** [comment]
A harvester configure as a remote harvester does not need to have the private key for the plots - the farmer that the remote harvester is connecting to, **does** need the private key for any plots that it is managing through local or remote harvesters

**Comment by BrandtH22** [comment]
@weiwenhao , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
